generators/app/index.js

```diff
@@ -124,42 +121,13 @@ module.exports = class extends BaseGenerator {
         }
     }
 
-    install() {
```

Comment:
This module doesn't change the Angular files -> so no need to launch yarn install or npm instal